### DISCUSSION TITLE
Command to return count and list of files analyzed by solargraph

### DISCUSSION THREAD

**Discussion by Fire-Dragon-DoL** [castwide/solargraph#673] [Ideas]
Hi,
My team encountered multiple times a problem where solagraph was analyzing more than 5000 files because there was something that wasn't excluded. One of the culprit was a nested gem with `vendor/bundle` directory for development gems.

Solargraph doesn't provide any feedback in what files it analyzed or how many it did through the command line (it does over socket though, given that VS Code informs us of that).

Would it be possible to provide a `list` command that lists all the files it intends to analyze, and maybe even a `--count` option that will just state the count? (or something along those lines. `wc -l` could be sufficient for counting)

This is very useful to debug setups as well as to validate that solargraph is properly configured.

**Comment by castwide** [comment]
The `list` command is on master and will be included in this month's release.
